sdk/runtime/runenv.go

```diff
@@ -204,93 +205,91 @@ func ParseRunEnv(env []string) (*RunEnv, error) {
 	return re, nil
 }
 
+// IsParamSet checks if a certain parameter is set.
+func (re *RunEnv) IsParamSet(name string) bool {
+	_, ok := re.TestInstanceParams[name]
+	return ok
+}
+
 // StringParam returns a string parameter, or "" if the parameter is not set.
-// The second return value indicates if the parameter was set.
-func (re *RunEnv) StringParam(name string) (s string, ok bool) {
+func (re *RunEnv) StringParam(name string) string {
 	v, ok := re.TestInstanceParams[name]
-	return v, ok
+	if !ok {
+		panic(fmt.Errorf("%s was not set", name))
+	}
+	return v
 }
 
-// StringParamD returns a string parameter, or the default value if not set.
-func (re *RunEnv) StringParamD(name string, def string) string {
-	s, ok := re.StringParam(name)
-	if !ok {
-		return def
+func (re *RunEnv) BytesParam(name string) uint64 {
```

Comment:
Let's call this `SizeParam`. In go, `BytesX` usually means "returns something as `[]byte`".